nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -25,6 +25,8 @@
 
 ## Backward Incompatibilities {#sec-release-24.11-incompatibilities}
 
+- `farm-haystack` has been deprecated since it will no longer receive updates and users are advised to upgrade to `haystack-ai`.
```

Comment:
We do not usually list the removal of packages, at least python packages, in the release notes.
Please add the alias to `pkgs/top-level/python-aliases.nix` instead.

Reply:
This shouldn't be aliased since the package name changes; it is in effect a new package even though it provides similar functionality. @JohnRTitor and @natsukium please reach consensus whether this should have release notes.

Reply:
I am fine with aliasing this, if the change is mostly backwards-compatible, else we should mention it in the release notes.

Reply:
Once again, the python module should not be listed in the nixos release notes, but in `pkgs/top-level/python-aliases.nix`. This is a leaf package, so there is no need to add it to the release notes.
I may have used the word "alias" incorrectly, but just throw an exception with the reason.